src/users/users.controller.ts

```diff
@@ -283,7 +283,18 @@ export class UsersController {
     @Req() req: FastifyRequest,
   ) {
     try {
-      const user = await this.usersService.updateUserInfo(req, email, body);
+      const originEmail = JSON.parse(
+        Buffer.from(
+          req.headers.authorization.split('.')[1],
+          'base64',
+        ).toString(),
+      ).user;
+
+      const user = await this.usersService.updateUserInfo(
+        originEmail,
```

Comment:
You don't need to pass `originEmail` into `usersService.updateUserInfo`, you can raise an exception here, where it is expected